guides/release/components/built-in-components.md

```diff
@@ -21,6 +21,23 @@ Will become:
 <label for="site">Ember Question</label>
 <input id="site" type="text" value="How do text fields work?"/>
 ```
+### Naming with aria-label
```

Comment:
I think maybe we want to remove this section and only show "naming with aria-labelledby. What do you think @MelSumner?

Reply:
fwiw, that is my understanding as well and I agree.

Reply:
I think it would be good to have "ways to associate labels" and then use/explain all of them.

Reply:
Little ping here for us to revisit this after we put up the Accessibility Spotlight blog posts perhaps. :)